src/com/magento/idea/magento2plugin/project/SettingsForm.form

```diff
@@ -63,7 +63,7 @@
               <forms/>
             </constraints>
             <properties>
-              <text value="Magento Installation Path"/>
+              <text value="Magento installation path:"/>
```

Comment:
Please remove the colon here and everywhere. We don't use it for form labels. Thanks

Reply:
Hi @VitaliyBoyko, thanks for your review. I have made those changes since most of the form labels in default PhpStorm settings uses a colon as shown in the screenshot below. Is there a reason that we're deviating from that design?

![Screenshot from 2020-10-20 18-24-34](https://user-images.githubusercontent.com/22623301/96588837-be16ea80-1301-11eb-94eb-09941818b8a3.png)